README.md

```diff
@@ -10,7 +10,7 @@
 This project is the App Page for Snowball (front-end) and a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
 ### Local: http://localhost:3000
-### Development: https://jupiter-frontend.vercel.app/
+### Development: https://app-v2-mocha.vercel.app/
```

Comment:
### Development: https://app-v2-git-development-snowballnetwork.vercel.app/

Reply:
@prodvdjin To clarify, is https://app-v2-mocha.vercel.app/ the preview server for the production branch, and https://app-v2-git-development-snowballnetwork.vercel.app/ is the preview server for the development branch?

If so, is there one for the candidate branch as well?

Reply:
@Ncookiez following is the preview server for candidate branch
https://app-v2-git-candidate-snowballnetwork.vercel.app/